pages/components/Logo/getDimensions.ts

```diff
@@ -0,0 +1,73 @@
+type dimension = string | undefined;
+type dimensions = [string, string];
+type measurementTransform = (m: number) => number;
+type transformApply = (s: string) => string;
+
+export const aspectRatio = 926 / 1065;
+
+function isMeasurementValid(str: string | undefined): Boolean {
+  const measurementRegex = /^\d+\.?\d*[a-zA-Z%]+$/;
+
+  if (str === undefined) {
```

Comment:
Could be less verbose if this value was stored in a variable, such as a `isStrUndefined` and then used to guard the regex check. Would remove an `if` statement and a `return` statement.